src/UserService.Validation/User/CreateUserRequestValidator.cs

```diff
@@ -12,30 +12,26 @@ public CreateUserRequestValidator()
             RuleFor(user => user.FirstName)
                 .NotEmpty()
                 .MaximumLength(32).WithMessage("First name is too long.")
-                .MinimumLength(1).WithMessage("First name is too short.")
-                .Matches("^[A-Z][a-z]+$|^[А-ЯЁ][а-яё]+$").WithMessage("First name with error.");
+                .MinimumLength(1).WithMessage("First name is too short.");
```

Comment:
what is this check for?  I guess you can delete it

Reply:
done